libp2p/NodeTable.h

```diff
@@ -297,7 +307,7 @@ class NodeTable : UDPSocketEvents
     std::shared_ptr<NodeSocket> m_socket;							///< Shared pointer for our UDPSocket; ASIO requires shared_ptr.
 
     // The info about PING packets we've sent to other nodes and haven't received PONG yet
-    std::unordered_map<NodeID, NodeValidation> m_sentPings;
+    std::map<bi::udp::endpoint, NodeValidation> m_sentPings;
```

Comment:
Switched to `map` here, because it was easier than creating a hash function for `udp::endpoint`